offix-offline/src/main/java/org/aerogear/offixoffline/InputTypeChecker.kt

```diff
@@ -0,0 +1,13 @@
+package org.aerogear.offixoffline
+
+/*
+Class to check whether the input parameter type of the receiver class of mutation matches to the Input<*> class of Apollo.
+ */
+class InputTypeChecker(val string: String) {
```

Comment:
can we have a simple function in the Offix file that does this checking for us instead of creating a new class?